homework-04/middlewares/verifyJWT.js

```diff
@@ -0,0 +1,26 @@
+import jwt from 'jsonwebtoken'
+import dotenv from 'dotenv'
+
+dotenv.config()
```

Comment:
I believe tihs should be placed at very index.js file

Reply:
resolved